user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/storage_per_DMA/Profile_migration_to_elastic.md

```diff
@@ -0,0 +1,103 @@
+
+When you [install an indexing database](xref:Installing_Elasticsearch_via_DataMiner) and [migrate booking data](xref:Configuring_DataMiner_Indexing) on an existing DataMiner System prior to DataMiner 10.4.0, profiles are not automatically migrated. Based on your specific setup, and keeping in mind the [limitations and differences with XML storage](#limitations-and-differences-with-xml-storage), you can decide independently whether or not you want to start this migration. However, **from DataMiner 10.4.0 onwards**, the use of an indexing database (or Storage as a Service) to store profiles is **mandatory**. Using XML storage is no longer possible from that DataMiner version onwards.
+
+> [!TIP]
+>
+> - If you are looking for information on how to migrate an SRM configuration from one DMA to another, see [Migrating an SRM configuration](xref:SRM_migrating).
+
+> [!NOTE]
+>
+> - XML storage is no longer supported from DataMiner 10.4.0 onwards. Upgrading to 10.4.0 will not be possible if the profile data is still stored in XML.
+> - Elasticsearch is only supported up to version 6.8. If you are using Elasticsearch as the indexing database, we therefore recommend using [Storage as a Service](xref:STaaS) instead, or if you do want to continue using self-hosted storage, using [dedicated clustered storage](xref:Dedicated_clustered_storage) with OpenSearch or the Amazon OpenSearch Service on AWS.
+
+## Migrating from XML to the indexing database
+
+To migrate the profiles, you will need to use the SLNetClientTest tool. Note that while the tool mentions only "Elastic", it can also be used for a migration to OpenSearch.
+
+![SLNetClientTest tool](~/user-guide/images/ClientTestToolMigrationUI_ProfileMigration.jpg)<br>
+*Migration window in SLNetClientTest tool (version 10.4.2)*
+
+> [!WARNING]
+>
+> - Always be extremely careful when using this tool, as it can have far-reaching consequences on the functionality of your DataMiner System.
+> - All Profile Manager instances in the cluster will keep working during the migration. But any creates/updates done during the migration will no longer be migrated.
+> - After migration the storage type will still be XML, to switch to indexing database storage, press the *Start using Elastic* button.
+
+1. Connect to the DMS using the *SLNetClientTest* tool:
+
+    1. In the *SLNetClientTest* tool, go to *Connection* > *Connect*.
+    1. In the *Connect* window, under *Authentication*, select *Explicit credentials* and specify the credentials of an administrator user.
+    1. Click the *Connect* button.
+
+1. In the *Advanced* menu, select *Migration*.
+
+1. In the *ProfileManger XML to Elastic* section, click the *Start Migration* button to start the migration wizard.
+
+   During the migrations, the Profile Manager is not stopped, so it will keep working as expected. This also means a snapshot of the Profiles.xml is migrated. Any creates/updates done during the migration will no longer be migrated.
```

Comment:
```suggestion
   During the migrations, the ProfileManager is not stopped, so it will keep working as expected. This also means a snapshot of the Profiles.xml is migrated. Any creates/updates done during the migration will no longer be migrated.
```